src/tools/tests/goldens/kotlin-connection-type-generator.cgtest

```diff
@@ -0,0 +1,49 @@
+[NAME]
```

Comment:
Could be nice if these had an auto-generated header re: "auto-generated via xxx command, do not manually edit".

Reply:
Good idea, added.

Note however that these are not entirely auto generated - we _can_ auto generate the part between `[RESULTS]` and `[END]` if we want to, but we don't even have to do that - you can write that by hand as well.

You are definitely expected to modify these files by hand, in particular add new test cases etc.